pkg/streaming/commands/processor_create_test.go

```diff
@@ -154,7 +172,9 @@ func TestProcessorCreateOptions(t *testing.T) {
 func TestProcessorCreateCommand(t *testing.T) {
 	defaultNamespace := "default"
 	processorName := "my-processor"
+	containerRef := "my-func"
```

Comment:
seems like this test string should be "my-container", for consistency / easy debugging